setup/linux/gui_solar_filter_setup.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/sh
+
```

Comment:
Lets rename this to `pipinstall.sh` so that we can all throw stuff into here

Also did you mean to put something in this empty script? :D

Reply:
Since tkinter is included in Python3, we actually don't need any imports for this